recipes/pyobjcryst/SConstruct.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/SConstruct b/SConstruct
+index 9b14d44..4c6301d 100644
+--- a/SConstruct
++++ b/SConstruct
+@@ -97,11 +97,6 @@ good_python_flags = lambda n : (
+ pyversion = pyoutput('import sys; print("%i.%i" % sys.version_info[:2])')
+ pythonconfig = 'python%s-config' % (pyversion if pyversion[0] == '3' else '')
+ # Verify python-config comes from the same path as the target python.
+-xpython = env.WhereIs(env['python'])
+-xpythonconfig = env.WhereIs(pythonconfig)
+-if os.path.dirname(xpython) != os.path.dirname(xpythonconfig):
+-    print("Inconsistent paths of %r and %r" % (xpython, xpythonconfig))
+-    Exit(1)
```

Comment:
@CJ-Wright - Is that change necessary?  That check prevents usage of incorrect python-config which could be tricky to track down when things go haywire.

Can you point me to a build log that failed because of the check?

Reply:
this one? https://circleci.com/gh/conda-forge/staged-recipes/22905?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

Reply:
I see.  Can you replace that patch with [0001-BLD-in-scons-convert-python-output-to-strings.patch.txt](https://github.com/conda-forge/staged-recipes/files/1679301/0001-BLD-in-scons-convert-python-output-to-strings.patch.txt) ?
(also at https://github.com/pavoljuhas/pyobjcryst/commit/35ec0dbd47c24ecb43c1fb8b16d256cb3f805ccf)

The check failed, because `"3" == b"3"` evaluated to False when checking Python version.